x/msgfee/init_test.go

```diff
@@ -0,0 +1,54 @@
+package msgfee
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/store"
+)
+
+func TestGenesis(t *testing.T) {
```

Comment:
Good happy-path test.
Adding a test with a good error value for an illegal fee would be good... we want to ensure invalid configs die early (on blockchain startup), not on first use. The code does this properly, but I like to see a test... which ensures later changes don't remove the validation accidentally

Reply:
This is tested alredy in `MsgFee.Validate` but ok.